consensus/src/simplex/elector.rs

```diff
@@ -0,0 +1,354 @@
+//! Leader election strategies for simplex consensus.
+//!
+//! This module provides the [`Elector`] trait for customizing how leaders are selected
+//! for each consensus round, along with built-in implementations.
+//!
+//! # Built-in Electors
+//!
+//! - [`RoundRobin`]: Simple deterministic rotation through participants based on view number.
```

Comment:
Would be nice for this to take some randomness on new (just once) to compute a permutation of leader order (just so it isn't the same neighbors in a stable peer set each epoch).